Description: Do not use gettid() for Bullseye and use syscall() instead
 This will downgrade the needed glibc version to 2.14 instead of 2.30 and avoid
 ordering issues during upgrades from Buster to Bullseye
 .
 This can be removed when Bullseye is released
Forwarded: not-needed
Origin: vendor
Bug-Debian: http://bugs.debian.org/979970
Author: Laurent Bigonville <bigon@debian.org>

--- a/src/procattr.c
+++ b/src/procattr.c
@@ -94,7 +94,7 @@ static int openattr(pid_t pid, const cha
 		if (fd >= 0 || errno != ENOENT)
 			goto out;
 		free(path);
-		tid = gettid();
+		tid = syscall(__NR_gettid);
 		rc = asprintf(&path, "/proc/self/task/%d/attr/%s", tid, attr);
 	} else {
 		errno = EINVAL;
